Ruby-Project/classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student(_student)
+    @student.push(Student)
+    Student.classroom = self
```

Comment:
The name for the parameter passed in this method should be the same throughout the entire method: 
```
def add_student(student)
    @student.push(student)
    student.classroom = self
end
```